modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -0,0 +1,271 @@
+
+|===
+| Option | Description
+
+| `enabled`
+| Whether or not to enable the `LoadBalancer` type of services integration. The default value is `true`.
+
+| `use-octavia`
+| Whether or not to use Octavia for the `LoadBalancer` type of service implementation rather than Neutron-LBaaS. The default value is `true`.
+
+| `floating-network-id`
+| Optional. The external network used to create floating IP addresses for load balancer virtual IP addresses (VIPs). If there are multiple external networks in the cloud, this option must be set or the user must specify `loadbalancer.openstack.org/floating-network-id` in the service annotation.
+
+| `floating-subnet-id`
+| Optional. The external network subnet used to create floating IP addresses for the load balancer VIP. Can be overridden by the service annotation `loadbalancer.openstack.org/floating-subnet-id`.
+
+| `floating-subnet`
+| Optional. A name pattern (glob or regular expression if starting with `~`) for the external network subnet used to create floating IP addresses for the load balancer VIP. Can be overridden by the service annotation `loadbalancer.openstack.org/floating-subnet`. If multiple subnets match the pattern, the first one with available IP addresses is used.
+
+| `floating-subnet-tags`
+| Optional. Tags for the external network subnet used to create floating IP addresses for the load balancer VIP. Can be overridden by the service annotation `loadbalancer.openstack.org/floating-subnet-tags`. If multiple subnets match these tags, the first one with available IP addresses is used.
+
+| `lb-method`
+| The load balancing algorithm used to create the load balancer pool. The value can be `ROUND_ROBIN`, `LEAST_CONNECTIONS`, or `SOURCE_IP`. The default value is `ROUND_ROBIN`.
+
+If you use the `ovn` provider, only the `SOURCE_IP` algorithm is supported.
+
+| `lb-provider`
+| Optional. Used to specify the provider of the load balancer, for example, `amphora` or `octavia`. Only the Amphora and Octavia providers are supported.
+
+| `lb-version`
+| Optional. The load balancer API version. Only `"v2"` is supported.
+
+| `subnet-id`
+| The ID of the Networking servie subnet on which load balancer VIPs are created. 
+
+// This ID is also used to create pool members if `member-subnet-id` is not set.
+
+// | `member-subnet-id`
+// | ID of the Neutron network on which to create the members of the load balancer. The load balancer gets another network port on this subnet. Defaults to `subnet-id` if not set.
+
+| `network-id`
+| The ID of the Networking service network on which load balancer VIPs are created. Unnecessary if `subnet-id` is set.
+
+// | `manage-security-groups`
+// | If the Neutron security groups should be managed separately. Default: false
+
+| `create-monitor`
+| Whether or not to create a health monitor for the service load balancer. A health monitor is required for services that declare `externalTrafficPolicy: Local`. The default value is `false`.
```

Comment:
perhaps we should mention that when using OVN driver monitor is only supported from OSP17

Reply:
Right, and there are several restrictions on the LoadBalancer section that were filed as document bugs: 

https://issues.redhat.com/browse/OCPBUGS-2217
https://issues.redhat.com/browse/OCPBUGS-2350
https://issues.redhat.com/browse/OCPBUGS-2789

Reply:
@maxwelldb will those restrictions be included in the release notes or in this docs?

Reply:
Likely both, as I read them now.